.gitignore

```diff
@@ -19,6 +19,14 @@ test/benchmark/baseline.txt
 xcuserdata/
 *.xccheckout
 
+# Allow Visual Studio project files.
+!wren/
+!wren_lib/
+
+# Visual Studio intermediate files.
+Debug/
+Release/
```

Comment:
Yeah, I can do that.

`wren.exe` and `wren*.lib` get sent to `build/vs/Debug` right now. Those should be in `bin/` and `lib/` right? Are debug and release separated (`bin/debug/`)?